webfriends-api/src/main/java/com/github/bordertech/webfriends/api/element/grouping/HStrong.java

```diff
@@ -0,0 +1,14 @@
+package com.github.bordertech.webfriends.api.element.grouping;
+
+import com.github.bordertech.webfriends.api.common.combo.FlowPalpableWithPhrasingElement;
+import com.github.bordertech.webfriends.api.common.tag.TagType;
+
+/**
+ * Paragraph element.
+ */
+public interface HStrong extends FlowPalpableWithPhrasingElement {
+
+	@Override
+	TagType getTagType();
```

Comment:
I think this copied an issue in HParagraph that should return TagParagraph and this can return TagStrong.

If you could update both, that would be great. Thanks.

This will also effect SParagraph and SStrong...see comments below for more detail....